libfwupdplugin/fu-self-test.c

```diff
@@ -5742,6 +5743,34 @@ fu_lzma_func(void)
 	g_assert_true(ret);
 }
 
+static void
+fu_cab_checksum_func(void)
+{
+	guint8 buf[] = {0x10, 0x20, 0x30, 0x40, 0x50, 0x60, 0x70, 80};
```

Comment:
is that last one supposed to be _0x80_?

Reply:
Ohh yes, fixed -- but that was just test data I was comparing against the old implementation and the new implementation. I've tested it with asan, valgrind, this local test data and in the end-to-end tests and I can't work out what the problem is with the new code...